src/OpenLayers/Controls/Drawing.js

```diff
@@ -18,6 +19,9 @@ var logger = Logger.getLogger("Drawing");
  * @param {Object} options - options for function call.
  * @param {Boolean} [options.collapsed = true] - Specify if Drawing control should be collapsed at startup. Default is true.
  * @param {ol.layer.Vector} [options.layer = null] - Openlayers layer that will hosts created features. If none, an empty vector layer will be created.
+ * @param {Object} [options.layerswitcher = {}] - ...
```

Comment:
Proposition : 
 * @param {Object} [options.layerDescription = {}] - Layer informations to be displayed in LayerSwitcher widget (only if a LayerSwitcher is also added to the map)
 * @param {Object} [options.layerDescription.title = "Croquis"] - Layer title to be displayed in LayerSwitcher
 * @param {Object} [options.layerDescription.description = "Mon croquis"] - Layer description to be displayed in LayerSwitcher